ui/app/pages/first-time-flow/seed-phrase/seed-phrase-intro/index.scss

```diff
@@ -0,0 +1,77 @@
+.seed-phrase-intro {
+  &__sections {
+    display: flex;
+
+    @media screen and (min-width: $break-large) {
+      flex-direction: row;
+    }
+
+    @media screen and (max-width: 830px) {
+      flex-direction: column;
+    }
+  }
+
+  &__left {
+    flex: 3;
+    min-width: 0;
+  }
+
+  &__right {
+    flex: 1;
+    min-width: 0;
+
+    @media screen and (max-width: 830px) {
+      margin-top: 24px;
+    }
+  }
+
+  &__title {
+    @include H1;
+
+    margin-bottom: 24px;
+    color: black;
+  }
+
+  &__title-copy {
+    max-width: 696px;
+    margin-bottom: 24px;
+  }
+
+  &__video-wrapper {
+    margin-bottom: 24px;
+
+    video {
+      border-radius: 8px;
+
+      @media screen and (max-width: 970px) {
+        width: 95%;
+      }
+    }
+  }
+
+  &__next-wrapper {
+    width: 325px;
+  }
+
+  &__sidebar {
+    padding: 15px;
+    border-radius: 8px;
+    border: 1px solid #d6d9dc;
```

Comment:
Don't use hex codes, match them to the color from the design system colors file. In this case $ui-2. https://github.com/MetaMask/metamask-extension/blob/12c614cc14fa9a146170e1989726deb758da310b/ui/app/css/design-system/colors.scss#L78-L110

Reply:
done